src/commands/install.yml

```diff
@@ -1,42 +1,105 @@
 description: "Install the AWS CLI via Pip if not already installed."
 
 parameters:
+  install-v2:
```

Comment:
What do you think of having a parameter like `install-version` instead, with an enum with values "1" and "2"? In the future, when there's a v3, we could then add "3" to the enum's possible values.

Reply:
Yes I was considering this but my reasoning for going this route at least for the moment is we can not use the `when` step on an enumerator, meaning we would need to lump all of the logic into a single bash statement which programmatically checks the value in a case switch or something.

With only 2 versions for the for the moment, allowing a boolean like this allowed me to keep the 1.0 source and the new 2.0 install in a separate command so the can be maintained separately. 

If "when" steps would eval against a value instead of evaluating as truthy that would satisfy both issues.

Reply:
@KyleTryon The limitation of the `when` step is indeed unfortunate! My preference is towards having a more future-proof parameter despite the internal implementation becoming more complex, but I understand the maintenance concern as well!

Reply:
Now installs as an enumerator. Even displays the selected version in the step name "dynamically":

https://app.circleci.com/pipelines/github/CircleCI-Public/aws-cli-orb/151/workflows/b04efaf5-4b1b-4e67-91ea-208218cf66e1/jobs/382

![image](https://user-images.githubusercontent.com/33272306/77679523-05eeb280-6f69-11ea-916c-4110b459aba6.png)